notifier.rst

```diff
@@ -51,6 +51,13 @@ The notifier component supports the following channels:
 SMS Channel
 ~~~~~~~~~~~
 
+.. caution::
```

Comment:
What's about using an include to not duplicate the content?

Reply:
I've merged this "as is" because the repeated content is in a single doc page, so the fragment wouldn't be reused in other pages and it feels a bit "too much". Thanks!